.ldrelease/publish-circleci.sh

```diff
@@ -3,8 +3,7 @@
 # Run this in publish step after all version information have been updated.
 set -ev
 
-sudo apt install curl
-./install-circleci
```

Comment:
@InTheCloudDan  I tested this in the [cimg/go](https://circleci.com/developer/images/image/cimg/go) container locally and it works. Turns out as a bonus that container already contains curl so we can remove the install curl command and execute the install directly.